src/vue/align-editor/align-editor.vue

```diff
@@ -36,45 +37,75 @@ export default {
   data () {
     return {
       defineAlignShow: false,
-      alignments: [],
-      currentAlignment: {},
-      alignmentId: 0,
-      prevClickedWordType: null
+      prevClickedWordType: null,
+      showAlignment: [],
+      originUpdated: 1,
+      targetUpdated: 1
     }
   },
   watch: {
     async showEditor () {

```

Comment:
Why is this function `async` if it does not use `await` inside?

Reply:
some edits before it had :) will fix

Reply:
added